packages/UI/src/context/UserDataContext.js

```diff
@@ -32,11 +32,11 @@ function useUserDataHook() {
   }, [userData.id > 0]);
 
   useEffect(() => {
-    axios(`${process.env.NEXT_PUBLIC_STRAPI_URL}/users/me?populate=deep`, {
+    axios(`${process.env.NEXT_PUBLIC_STRAPI_URL}/users/80?populate=deep`, {
```

Comment:
please revert this, we should have this call /users/me not /users/80

Reply:
Fixed it, thanks for catching that.